api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -730,4 +731,73 @@ public boolean isOrderTypeInUse(OrderType orderType) {
 		criteria.add(Restrictions.eq("encounter", encounter));
 		return criteria.list();
 	}
+
+	/**
+	 * @see org.openmrs.api.db.OrderDAO#getOrderGroupAttributeByUuid(String)
+	 */
+	@Override
+	public OrderGroupAttribute getOrderGroupAttributeByUuid(String uuid)  throws DAOException{
+		if(uuid == null || uuid.isEmpty()){
```

Comment:
I would remove these checks for the null or empty parameter to throw an exception.